README.md

```diff
@@ -134,8 +134,34 @@ Here is an example of how to provide entry point for each plugin type:
 We have developed two distinct Data Source plugins, each serving a specific purpose:
 
 1. **HTML Data Source:**
-   - Description: This plugin is designed to retrieve data from HTML documents.
+   - Description: This plugin is designed to retrieve data from HTML documents and represent it as a cyclic and acyclic graph.
```

Comment:
A graph can't be both cyclic and acyclic. If you mean that graph could be either cyclic or acylic, you can just say graph.
```suggestion
   - Description: This plugin is designed to retrieve data from HTML documents and represent it as a graph.
```